crates/core/src/factories/core_fetch.rs

```diff
@@ -97,6 +98,9 @@ pub mod config {
     pub struct CoreFetchConfig {
         /// How many parallel op fetch requests can be made at once. Default: 2.  
         pub parallel_request_count: u8,
+        /// Delay before re-inserting ops to request back into the outgoing request queue.
+        /// Default: 100 ms.
+        pub re_insert_outgoing_request_delay: Duration,
```

Comment:
Don't we need to be careful that these can deserialize from YAML and use `_ms` or is that not a concern here?

Reply:
I do not know. I remember there was something yamly. Durations can be deserialized from YAML, but it's not so user-friendly.
I'll just convert all to ms.